homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -40,24 +41,6 @@ class AccSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         value_fn=AladdinConnectClient.get_battery_status,
     ),
-    AccSensorEntityDescription(
```

Comment:
Why are these sensors being removed?

Reply:
Not all devices support this value, and there is no model number currently included in our partner API, therefore it requires modification of our API to support knowing whether or not to include this value.